devtools-ci-index.yaml

```diff
@@ -2191,6 +2201,16 @@
             kc_refresh_token_creds: 807b706e-c54f-48d5-be93-56525cf499f0
             ee_test_start_time: '5 2-23/4 * * *'
             timeout: 60m
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
+            test_url: openshift.io
+            test_suite: logintest
+            cluster: cluster-two
+            after: devtools-fabric8-test-build-master
+            osio_creds: e9a32e1e-0a38-4a96-896f-a266e1428cf9
+            oso_token_creds: de0940f2-f31f-4186-85e2-37d42bddcf5b
+            kc_refresh_token_creds: 807b706e-c54f-48d5-be93-56525cf499f0
+            ee_test_start_time: '*/15 * * * *'
+            timeout: 60m
```

Comment:
Why does cluster-two have a timeout of 60min when cluster-one only has 10m?

Reply:
EXCELLENT question! I just changed the timeout for both tests to 5 min. The test should run in only 30 sec - so if it ever requires 5 min, we want it to fail. Thanks!